config/sequelize.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable indent */
+const dotenv = require('dotenv').config()
+
+module.exports = {
+    development: {
+        host: process.env.DB_HOST,
+        database: process.env.DB_DATABASE,
```

Comment:
Your .env sample doesn't have a `DB_DATABASE` in it. When I copied .env_sample to make my .env file to run this I ended up with an error because of this.